src/app/pages/system/general-settings/email/email-card/email-card.component.html

```diff
@@ -0,0 +1,40 @@
+<mat-card id="email-card">
+  <mat-toolbar-row>
+    <h3>{{ 'Email' | translate }}</h3>
+    <div class="actions action-icon">
+      <button
+        *ixWithLoadingState="emailConfig$ as emailConfig"
+        mat-button
+        color="default"
+        [ixTest]="['email', 'settings']"
+        (click)="openEmailSettings(emailConfig)"
+      >
+        {{ 'Settings' | translate }}
+      </button>
+    </div>
+  </mat-toolbar-row>
+
+  <mat-card-content>
+    <mat-list>
+      <mat-list-item>
+        <span class="label">{{ 'Send Mail Method' | translate }}:</span>
+        <span *ixWithLoadingState="emailConfig$ as emailConfig" class="value">
+          {{ !emailConfig?.oauth?.client_id ? helptext.send_mail_method.smtp.placeholder : helptext.send_mail_method.gmail.placeholder }}
+        </span>
+      </mat-list-item>
+
+      <ng-container *ixWithLoadingState="emailConfig$ as emailConfig">
+        <ng-container *ngIf="!emailConfig?.oauth?.client_id">
+          <mat-list-item>
+            <span class="label">{{ 'From' | translate }}:</span>
+            <span *ixWithLoadingState="emailConfig$ as emailConfig" class="value">
+              {{ emailConfig.fromname }}
+              {{ emailConfig.fromemail || '-' }}
+              {{ emailConfig.outgoingserver ? (' via ' | translate) + emailConfig.outgoingserver : ''}}
```

Comment:
Just translating `via` is not a good idea, because in different languages the order of words may not be the same.
It should be `{email} via {server}` when both parts are set.